### DISCUSSION TITLE
No auto update

### DISCUSSION THREAD

**Discussion by Robbie-65** [Koenkk/zigbee2mqtt#15429] [General]
Hello,
I will probably do something wrong here, but my zigbee2mqtt still shows version 1.18 although auto update is on.
How can I install the latest release?

**Comment by Koenkk** [comment]
If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Reply by Robbie-65** [reply]
Hi Koen,
Thanks, I had already seen the Edge version, but thought that was some sort of Beta version.
So what you mean is, I should switch to Edge? and the regular version will stay on 1.18?
I will look into that.
What is the easiest way to copy my existing config to the Edge version?

Thanks,

Rob

**Reply by Koenkk** [reply]
You need to switch to the new repository (not edge, the release version).
